### PR TITLE
[DNS] Added subdomains page + revised proxy status page

### PR DESCRIPTION
**Pull request by kodster28**
Finishes PCX-3229

### CODE REVIEW

File: products/dns/src/content/manage-dns-records/how-to/create-subdomain.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (kodster28) [inline products/dns/src/content/manage-dns-records/how-to/create-subdomain.md:1]: This is a tad duplicative of what's in Fundamentals, but going to trim down the fundamentals page once this page goes live.

File: products/dns/src/content/manage-dns-records/how-to/create-subdomain.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Subdomain records
+
+To create a new subdomain, you would first add the subdomain content at your host.
+
+Then, you would create a corresponding [A, AAAA, or CNAME record](/manage-dns-records/how-to/create-dns-records) for that subdomain (`blog`, `store`).
+
+<Example>
+
+| **Type** | **Name** | **IPv4 address** | **Proxy status** |
```
Comment (pedrosousa) [inline products/dns/src/content/manage-dns-records/how-to/create-subdomain.md]:
```suggestion
| Type | Name | IPv4 address | Proxy status |
```
Table column headings already have bold formatting (CSS).

File: products/dns/src/content/manage-dns-records/how-to/create-subdomain.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+| --- | --- | --- | --- |
+| A | `www` | `192.0.2.1`| Proxied |
+
+</Example>
+
+## Subdomain redirects
+
+For more guidance on redirecting a subdomain — either to your main domain or another location — refer to [Subdomain redirects](https://developers.cloudflare.com/fundamentals/get-started/basic-tasks/manage-subdomains#set-up-redirects).
```
Comment (pedrosousa) [inline products/dns/src/content/manage-dns-records/how-to/create-subdomain.md]: Nit: "Subdomain redirects" currently doesn't match any section name in the linked Fundamentals page (but the link is correct).

File: products/dns/src/content/manage-dns-records/reference/proxied-dns-records.md [hunk 4]
```diff
@@ -5,21 +5,27 @@ pcx-content-type: concept
...
 
-Beyond the performance and caching benefits, proxying your records hides your server's IP address and protects your application from [DDoS attacks](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/).
+Beyond the performance and caching benefits, proxying your records hides your origin server's IP address and protects your application from [DDoS attacks](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/).
 
 ## When to use unproxied records
 
 In some circumstances, you should not proxy your DNS records.
 
-If you need to connect to your origin using a non-HTTP protocol (SSH, FTP, SMTP) or over a different port than ports 80 or 443, either leave your records unproxied (DNS-only) or use [Cloudflare Spectrum](https://developers.cloudflare.com/spectrum).
+If you need to connect to your origin using a non-HTTP protocol (SSH, FTP, SMTP) or destined for an [unsupported port](https://developers.cloudflare.com/fundamentals/get-started/network-ports), either leave your records unproxied (DNS-only) or use [Cloudflare Spectrum](https://developers.cloudflare.com/spectrum).
```
Comment (pedrosousa) [inline products/dns/src/content/manage-dns-records/reference/proxied-dns-records.md]: I believe that the second part of the sentence no longer follows its beginning.

Before: `If you need to connect to your origin ... over a different port than ports 80 or 443`
After: `If you need to connect to your origin ... destined for an unsupported port`

Maybe change to something like the following:
```
If you need to connect to your origin using a non-HTTP protocol (SSH, FTP, SMTP) or the traffic targets an [unsupported port](...) at the origin,
```

File: products/fundamentals/src/content/get-started/basic-tasks/manage-subdomains.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ For deeper subdomains (`dev.blog.example.com`), use a [different type of certifi
 
 ## Customize subdomain behavior
 
-If you want custom Cloudflare settings for individual subdomains, your approach will vary depending on your plan.
+If you want customize Cloudflare settings for individual subdomains, your approach will vary depending on your plan.
```
Comment (pedrosousa) [inline products/fundamentals/src/content/get-started/basic-tasks/manage-subdomains.md]:
```suggestion
If you want to customize Cloudflare settings for individual subdomains, your approach will vary depending on your plan.
```
(I understand that you will later review this duplicated content in Fundamentals.)

File: products/dns/src/content/manage-dns-records/how-to/create-subdomain.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## SSL/TLS for subdomains
+
+If your main domain is using Cloudflare's [Universal SSL certificate](https://developers.cloudflare.com/ssl/edge-certificates/universal-ssl), that certificate also covers all first-level subdomains (`blog.example.com`). 
+
+For deeper subdomains (`dev.blog.example.com`), use a [different type of certificate](https://developers.cloudflare.com/ssl/edge-certificates/universal-ssl/limitations#full-setup).
+
+## Customize subdomain behavior
+
+If you want customize Cloudflare settings for individual subdomains, your approach will vary depending on your plan.
```
Comment (pedrosousa) [inline products/dns/src/content/manage-dns-records/how-to/create-subdomain.md]:
```suggestion
If you want to customize Cloudflare settings for individual subdomains, your approach will vary depending on your plan.
```

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**